documents/entity/service.go

```diff
@@ -220,3 +220,65 @@ func (s service) DeriveFromUpdatePayload(ctx context.Context, payload *clientent
 
 	return entity, nil
 }
+
+func (s service) GetVersion(ctx context.Context, documentID []byte, version []byte) (documents.Model, error) {
+	return s.get(ctx, documentID, version)
+}
+
+// isDIDaCollaborator returns true if the did is a collaborator of the document
+func (s service) isDIDaCollaborator(ctx context.Context, did identity.DID, model documents.Model) (bool, error) {
```

Comment:
```suggestion
func (s service) isDIDCollaborator(ctx context.Context, did identity.DID, model documents.Model) (bool, error) {
```

Reply:
We can move this function to model itself

Reply:
First I had it there. Afterwards, I realized I would need to add it to the interface and for me, it is more a helper function.

I agree the method should be re-usable for all documents.